### ISSUE TITLE
static HBITMAP leaked in explorer process

### ISSUE THREAD

**Issue by chrdavis** [sanje2v/PellucidIcons#1]
Every time the module is loaded the HBITMAP is leaked.  Please make the static HBITMAP used in PellucidHandlers::QueryContextMenu a member of PellucidHandlers so you can call DeleteObject on it in ~PellucidHandlers.

**Comment by sanje2v** [comment]
Hi @chrdavis , thanks for your bug report. I have pushed an update to delete the leaking bitmap handle in the class destructor when the reference count for COM object is 0 or less.

The reason I think I made the bitmap handle static was probably to share the same bitmap resource in the multiple instances of the COM object. So, I have left it as a static variable.

On a different note, I can see that you are part of Edge Chromium team at Microsoft. Would be great if you guys could port 'bookmark tags' feature from Mozilla Firefox in coming iterations. :)
